docs/releng.adoc

```diff
@@ -0,0 +1,79 @@
+:showtitle:
+:numbered:
+:toc: left
+
+= Oxide Release Engineering
+
+Omicron is the Oxide control plane, and thus brings together all of the
+various components outside of this repo that make up the software on the
+product. This includes (but definitely isn't limited to):
+
+- https://github.com/oxidecomputer/propolis[Propolis], our hypervisor
+- https://github.com/oxidecomputer/helios[Helios], our host operating
+  system
+- https://github.com/oxidecomputer/crucible[Crucible], our block storage
+  service
+- https://github.com/oxidecomputer/maghemite[Maghemite], our switch
+  control software and routing protocol
+- https://github.com/oxidecomputer/hubris[Hubris], our embedded
+  microcontroller operating system used on the root of trust and service
+  processors
+- https://github.com/oxidecomputer/console[The web console]
+
+Each of these has their own build processes that produce some sort of
+usable artifact, whether that is an illumos zone or a tarball of static
+assets.
+
+The release engineering process builds the control plane and combines
+it with the many external artifacts into a final artifact -- a Zip
+archive of a TUF repository -- that contains everything necessary for
+the product to operate. This process is run on each commit to ensure it
+is always functional. You can also run the process locally with `cargo
+xtask releng`.
```

Comment:
this wrapping doesn't render nicely

Reply:
Where?

Reply:
(I fixed it about three seconds before David posted his note)